### ISSUE TITLE
Only one letter shown as title when creating a window on Windows.

### ISSUE THREAD

**Issue by Kidsunbo** [rougier/pyglfw#9]
Only one letter shown as title when creating a window on Windows. Except that, everything works fine. If I deleted the explicit title parameter, only a `G` is shown at the position of the title.
How to solve that?
The system is Windows 10 and with Python 3.7.3. The version of GLFW is 3.3.

**Comment by rougier** [comment]
Might be a string encoding problem. Can you post a minimal example showing the problem?

**Comment by Kidsunbo** [comment]
```Python
from glfw import *
import OpenGL.GL as gl

def main():
    glfwInit()
    glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR,3)
    glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR,3)
    
    window = glfwCreateWindow(640,480) # I use the default parameter, but only one letter is shown
    
    glfwMakeContextCurrent(window)
    
    while not glfwWindowShouldClose(window):
        glfwPollEvents(window)
        gl.glClearColor(1,0,0,1)
        gl.glClear(gl.GL_COLOR_BUFFER_BIT)
        glfwSwapBuffers(window)
    glfwDestroyWindow(window)
    glfwTerminate()

if __name__ == '__main__':
    main()
```

Yeah sure. Here is the sample code on my machine.
After running this code, a red window would be shown. But the title of the window is ‘G'.
![20191126142320](https://user-images.githubusercontent.com/12459780/69604419-59530180-1058-11ea-8f63-d0436a4c9c3d.png)

**Comment by rougier** [comment]
Can you try to set the title explicitly to test if this make any difference ?

**Comment by Kidsunbo** [comment]
> Can you try to set the title explicitly to test if this make any difference ?

Yeah I've tried to set the title explicitly. I used "GLFW Window" as parameter and only one letter 'G' was shown. If I used other words, the shown title was still the first letter of the words.

**Comment by rougier** [comment]
Can you try setting the title to `b"test"` or `"test".encode("UTF8")` or `"test".encode("ASCII")`

**Comment by Kidsunbo** [comment]
> Can you try setting the title to `b"test"` or `"test".encode("UTF8")` or `"test".encode("ASCII")`

It works. Thanks so much.

**Comment by rougier** [comment]
I'm reopening it because it still needs to be fixed in sources. Do you want to try a PR ?

**Comment by Kidsunbo** [comment]
> I'm reopening it because it still needs to be fixed in sources. Do you want to try a PR ?

Sure.
